SIVA/Core/Discord/RawEmoji.cs

```diff
@@ -0,0 +1,878 @@
+﻿namespace SIVA.Core.Discord {
+    
+    public class RawEmoji {
+        
+        public readonly string A = "\uD83C\uDD70";
```

Comment:
Make these `static` or `const`?

Reply:
static uses more memory if I understand it correctly, and I have like no ram left on my VPS. tbh instantiation doesn't bother me too much

Reply:
Well then replace with
```cs
public static string A => "\uD83C\uDD70"
```
This is static, only puts the bytes in memory when needed and uses less memory. Heck even static uses less memory than giving the GC a hard time.

Also why not use `const` like i suggested before?